etc/schema.json

```diff
@@ -6017,36 +6018,51 @@
                     "type": "object",
                     "properties": {
                         "active": {
+                            "description": "Number of currently active flows",
                             "type": "integer"
                         },
                         "emerg_mode_entered": {
+                            "description": "Number of times emergency mode was entered",
                             "type": "integer"
                         },
                         "emerg_mode_over": {
+                            "description": "Number of times recovery was made from emergency mode",
                             "type": "integer"
                         },
                         "get_used": {
+                            "description": "Number of reused flows",
                             "type": "integer"
                         },
                         "get_used_eval": {
+                            "description":
+                                    "Number of attempts at getting a flow directly from the hash",
                             "type": "integer"
                         },
                         "get_used_eval_busy": {
+                            "description":
+                                    "Number of times a flow was found in the hash but the lock for hash bucket could not be obtained",
                             "type": "integer"
                         },
                         "get_used_eval_reject": {
+                            "description":
+                                    "Number of flows that were successfully retrieved from hash but were still alive",
```

Comment:
can you explain what this means?

Reply:
In case we have reached the memcap for the flows and the spare queue is empty, we try to retrieve a flow directly from the hash table.
However, this retrieved flow might still be "alive", so we end up rejecting such a flow for use and increment this counter.

This is what I got. wdyt?

Reply:
Guess I'm tripping over "retrieved" here, maybe something like `Number of flows that were evaluated but rejected from reuse as they were still alive/active`